src/main/scala/App.scala

```diff
@@ -17,8 +17,8 @@ import zio.http.netty.client.NettyClientDriver
 import java.net.URI
 import java.net.http.{HttpClient, HttpRequest, HttpResponse}
 
-
-val urlHttp = "http://google.com"
+//for now, ensure not http requests are used
```

Comment:
The http tests were there to check for https overhead.  We should leave that in.

Reply:
can be added later, currently just too much.